src/include/simeng/config/SimInfo.hh

```diff
@@ -0,0 +1,245 @@
+#pragma once
+
+#include <iostream>
+#include <string>
+
+#include "simeng/Instruction.hh"
+#include "simeng/RegisterFileSet.hh"
+#include "simeng/config/ModelConfig.hh"
+#include "simeng/config/yaml/ryml.hh"
+
+namespace simeng {
+namespace config {
```

Comment:
Can this be named something else? In many places `config = SimIngo::getConfig()` and then `config::simMode` or `config::ISA` is used which is confusing.

I'll leave this decision up to you about which identifier to rename i.e variable or namespace

Reply:
I think that the choices for both variable names and namespace makes sense and are optimal choices. Not sure I truly understand the source of your confusion. If it is that prevalent for you, then I'd ask that you provide a pair of different names and then we can discuss their suitability.